### PR TITLE
DEPRECATED feat: Truera integration, check https://github.com/newrelic/newrelic-quickstarts/pull/566

### PR DESCRIPTION
**Pull request by vivekbhalgat**
# Summary

Create a PR for Truera integration with New Relic.

As truera wont be creating dashboard using UI, I have not added dashboard folder under quickstart/truera/
OPEN QUESTION: How to generate id used in quickstarts/truera/config.yml?

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work? NA
- [ ] Did you check your dashboard image quality? -  Do they look good?  NA
- [ ] Did you check that your alerts actually work?  NA
- [ ] Did you include an InstallPlan and Documentation reference?  Yes(The links need to be updated once I get final version from @iazam)
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?  Yes
- [ ] Did you attach images of your dashboards to the PR so we can see them working?  NA, No dashbards are created, there would be a different tool to setup dashboards

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/mlops/truera/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+id: d0e829a6-ede4-4933-9065-9f0c56f8aa7e # TODO(vivek): How to generate?
```
Comment (rudouglas) [inline quickstarts/mlops/truera/config.yml:1]: Please remove this `id` as these will get auto-generated when the PR is merged
```suggestion
```

File: quickstarts/mlops/truera/dashboards/truera.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
+{
+  "name": "Truera: Initial Dashboard",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```
Comment (rudouglas) [inline quickstarts/mlops/truera/dashboards/truera.json:4]: If you could also remove the permissions parameter that would be great too. This parameter is in the json when you copy a dashboard, but it causes issues when you go to create a Dashboard using JSON
```suggestion
```

**Review by rudouglas — DISMISSED** [review]
Just a couple of things need to be addressed, otherwise looks great 🥳

**Comment by rudouglas** [comment]
@vivekbhalgat i've left some comments with changes that need to be made for this to pass validation, if you could make those as soon as you get a chance that would be great :D

File: quickstarts/mlops/truera/dashboards/truera.json (hunk 2 above)
Comment (vivekbhalgat) [reply quickstarts/mlops/truera/dashboards/truera.json:4]: removed description field based on comment in another PR and permission field based on comment here.

File: quickstarts/mlops/truera/config.yml (hunk 1 above)
Comment (vivekbhalgat) [reply quickstarts/mlops/truera/config.yml:1]: I ran into a few issues while exporting fixes for comments, created new PR instead. Please review it.
https://github.com/newrelic/newrelic-quickstarts/pull/566

**Comment by vivekbhalgat** [comment]
> Just a couple of things need to be addressed, otherwise looks great partying_face

Resolved.

**Comment by jpvajda** [comment]
@vivekbhalgat do you mind signing the [CLA agreement](https://github.com/newrelic/newrelic-quickstarts/pull/537#issuecomment-957607313)? This is required for new contributors to this repo.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**